### DISCUSSION TITLE
Feature freeze starting 1 December 2022

### DISCUSSION THREAD

**Discussion by emckean** [google/docsy#1308] [Announcement]
Hello Docsytarians! 

First: a huge thanks to everyone who has contributed to Docsy this year! We've seen a remarkable number of helpful PRs adding and updating features and fixing bugs! 

In order to migrate to the new Bootstrap version 5 (#470), we're declaring a feature freeze beginning **1 December 2022**. 

**What does this mean?** 

- You can still fork the project and make pull requests during the freeze: we just won't be merging any feature PRs until the Bootstrap merge is complete 
- We'll still be merging PRs for security issues or serious bugfixes

**When will the freeze be over?** 

We hope we'll be able to finish the Bootstrap merge by end of ~January~ February. 

Questions? Please ask them here in this discussion.

**Comment by H20220009** [comment]
@emckean 
Is the Bootstrap merge over, OR is the Freeze over?

**Reply by emckean** [reply]
Hi, we'll announce here when the merge is complete and the freeze is lifted.

**Comment by mahi072** [comment]
Hi @emckean , is this Freeze over?
I have raised the PR(#1488), please review it.

**Comment by chalin** [comment]
With the following:

- #1555
- #470 completed

We declare the feature freeze over. Note that it will take some time to make our way through the backlog of PRs.

**Reply by deining** [reply]
> Note that it will take some time to make our way through the backlog of PRs.

**FYI**: I just rebased all my  pending [pull requests](https://github.com/google/docsy/pulls/deining) to HEAD of repo and resolved all merge conflicts that were present. Hope that makes review easier for you :smile:.

**Reply by chalin** [reply]
Yes, thanks!
